### DISCUSSION TITLE
How to cite Quarto?

### DISCUSSION THREAD

**Discussion by lorenzoFabbri** [quarto-dev/quarto-cli#10475] [Q&A]
### Description

Is there a correct way to cite Quarto in our works? The same way I would cite the paper corresponding to a specific R package. Thank you.

**Comment by mcanouil** [comment]
Use the citation from the "Cite this repository"
<img width="250px" src="https://github.com/user-attachments/assets/0d0558b3-1d6f-443e-8196-3de1f9656e53" />

Duplicate of:

- https://github.com/quarto-dev/quarto-cli/discussions/4366
